Tokens/tokentofriend/clients/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.8.5'
 
     // Corda dependencies.
+    implementation "$corda_release_group:corda-core:$corda_release_version"
```

Comment:
what is the need to add corda-core here?

Reply:
Something must have changed between 4.11 and 4.12 around corda-core, I had to add explicit dependency for it in quite a few CorDapps. Without it the `Client` won't compile:
![image](https://github.com/corda/samples-java/assets/82059118/a753fbd9-970a-4465-9394-770c32a7c273)